test/libsolidity/syntaxTests/pragma/empty_version.sol

```diff
@@ -1,3 +1,3 @@
 pragma solidity;
```

Comment:
Can you also add this: `pragma solidity ;` (i.e. a space after `solidity`)? Should trigger the same message.